bnd.bnd

```diff
@@ -3,6 +3,7 @@ Export-Package: !org.junit.platform.*, \
                 junit.*;version=${junit.version}, \
                 org.junit.*;version=${junit.version}, \
                 org.hamcrest.*;version=${hamcrest.version};-split-package:=merge-first
+Sling-Test-Regexp: .*JTest
```

Comment:
I think this is not needed. As this PR has been merged already I have removed it in commit 88732dd. @cmrockwell please cross-check!

Reply:
Agreed. The tests were moved to annotations-it. This line is present in src/it/annotations-it/bnd.bnd